src/shared/components/error/Error.css

```diff
@@ -3,7 +3,6 @@
 
 .errorScreen {
     width: 100%;
-    height: 100%;
```

Comment:
Why was this removed?

Reply:
See the note on `display: flex` above. Both in the case where this `Error` component is used in `Boot` and the case where it is used in `App.page`, the parent component has an attribute `flex: 1 0 auto` and the `height: 100%` does not affect it because the parent does not have a specified height

Reply:
Lets keep the height because we might not know if the parent is flexed.

Reply:
👍